mermithid/processors/misc/MultiGasComplexLineShape.py

```diff
@@ -0,0 +1,3089 @@
+'''
+Fits data to complex lineshape model.
+Author: E. Machado, Y.-H. Sun, E. Novitski, T. Weiss, X. Huyan
+Date: 2/9/2021
+
+This processor takes in frequency data in binned histogram and fit the histogram with two gas scattering complex line shape model.
```

Comment:
please add some description of the model. what is included? track reconstruction efficiency, efficiency vs frequency (?), instrumental resolution functions, ...